lib/tasks/fetch/buildings.rake

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+namespace :fetch do
+  desc 'Fetch buildings from Meksis'
+  task buildings: :environment do
+    buildings = Xokul::Meksis.buildings
+
+    progress_bar = ProgressBar.spawn 'Meksis - Buildings', buildings.size
+
+    buildings.each do |building|
+      place_type = PlaceType.find_by(meksis_id: building[:place_type_id])
+      unit = Unit.find_by(yoksis_id: building[:yoksis_id])
+
```

Comment:
Bu task sadece `create` yapıyor, hem create ve update birlikte yapılsa iyi olur.

Örneğin:

```ruby
  building = Building.find_or_initialize_by(meksis_id: building[:id])
  building.assign_attributes(...)
  building.save

```